### DISCUSSION TITLE
PaddleOCR 训练竖排的长方形图片时参数怎么调呢？哪位大神帮看一下

### DISCUSSION THREAD

**Discussion by account15222** [PaddlePaddle/PaddleOCR#16444] [Q&A]
我的训练图片都是竖排的 高320 宽48 彩色的

这是训练参数
Global:
  debug: false
  use_gpu: true
  epoch_num: 300
  log_smooth_window: 20
  print_batch_step: 10
  save_model_dir: ./output/v3_ganzhur7
  save_epoch_step: 3
  eval_batch_step: [0, 100]
  cal_metric_during_train: true
  pretrained_model: ./output/v3_base/best_accuracy
  checkpoints:
  save_inference_dir:
  use_visualdl: true
  infer_img: doc/imgs_words/ch/word_1.jpg
  character_dict_path: ppocr/utils/dict/mong_dict.txt
  max_text_length: &max_text_length 60
  infer_mode: false
  use_space_char: true
  distributed: true
  save_res_path: ./output/rec/predicts_ganzhur_ppocrv3.txt

Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    name: Cosine
    learning_rate: 0.00005  # 调小 10 倍
    warmup_epoch: 2
  regularizer:
    name: L2
    factor: 3.0e-05

Architecture:
  model_type: rec
  algorithm: SVTR
  Transform:
  Backbone:
    name: MobileNetV1Enhance
    scale: 0.5
    last_conv_stride: [1,2]
    last_pool_type: avg
    #freeze: true  # 冻结 backbone
  Head:
    name: MultiHead
    head_list:
      - CTCHead:
          Neck:
            name: svtr
            dims: 64
            depth: 2
            hidden_dims: 120
            use_guide: True
          Head:
            fc_decay: 0.00001
      - SARHead:
          enc_dim: 512
          max_text_length: *max_text_length

Loss:
  name: MultiLoss
  loss_config_list:
    - CTCLoss:
    - SARLoss:

PostProcess:  
  name: CTCLabelDecode

Metric:
  name: RecMetric
  main_indicator: acc
  ignore_space: False

Train:
  dataset:
    name: SimpleDataSet
    data_dir: ./train_data/rec
    ext_op_transform_idx: 1
    label_file_list:
      - ./train_data/rec/v3_train.txt
    transforms:
      - DecodeImage:
          img_mode: BGR
          channel_first: false
      - RecAug:
          prob: 0.3
          image_shape: [48, 320]
      - MultiLabelEncode:
      - RecResizeImg:
          image_shape: [3,48,320]
      - KeepKeys:
          keep_keys:
          - image
          - label_ctc
          - label_sar
          - length
          - valid_ratio
  loader:
    shuffle: true
    batch_size_per_card: 32  # 减小 batch size
    drop_last: true
    num_workers: 4

Eval:
  dataset:
    name: SimpleDataSet
    data_dir: ./train_data/rec
    label_file_list:
      - ./train_data/rec/v3_test.txt
    transforms:
      - DecodeImage:
          img_mode: BGR
          channel_first: false
      - MultiLabelEncode:
      - RecResizeImg:
          image_shape: [3, 48, 320]
      - KeepKeys:
          keep_keys:
          - image
          - label_ctc
          - label_sar
          - length
          - valid_ratio
  loader:
    shuffle: false
    drop_last: false
    batch_size_per_card: 32
    num_workers: 4
